src/libmain/progress-bar.cc

```diff
@@ -408,7 +408,11 @@ class ProgressBar : public Logger
             }
         }
 
+#ifndef _WIN32
         auto width = getWindowSize().second;
+#else
+        auto width = 0;
+#endif
```

Comment:
https://stackoverflow.com/questions/6812224/getting-terminal-size-in-c-for-windows has something for this

Reply:
#10543